index.html

```diff
@@ -1879,6 +1879,88 @@ <h3>Versioning Cryptography Suites</h3>
+particular application developer might need for a given cryptographic algorithm.
+All options are often exposed to application developers.
+        </p>
+        <p>
+Application developers that use cryptographic libraries often do not have the
+same level of knowledge as cryptographic algorithm designers, specification
+authors, or cryptographic library implementers. This lack of knowledge can lead
+to an inappropriate selection of cryptographic parameters and options for a
+particular application.
+        </p>
+        <p>
+This specification sets the priority of constituencies to protect application
+developers over cryptographic library implementers over cryptographic
+specification authors over cryptographic algorithm designers. Given these
+priorities, the following recommendations are made:
+        </p>
+        <ul>
+          <li>
+Cryptographic algorithm designers are advised to minimize the number of
+options and parameters to as few as possible to ensure that
+cryptographic library implementers have a more easily auditable security
+attack surface for their software libraries.
+          </li>
+          <li>
+Cryptographic specification authors are advised to, if possible, further
+minimize the number of options and parameters to as few as possible to ensure
+cryptographic agility while also keeping the auditable security
+attack surface for downstream software libraries to a minimum.
+          </li>
+          <li>
+Cryptographic library implementers are advised to, if possible, provide known
+good combinations of options and parameters to application developers. By
+default, there would ideally be two pre-set configurations for any algorithmic
+class, such as Elliptic Curve Digital Signatures, with no ability (by default)
+to fine tune parameters and options. While library options can be provided to
+experts to fine tune the library, usage of those options among the general
+application developer population is to be discouraged.
+          </li>
+          <li>
+Application developers are advised to choose from a number of pre-set
+cryptography library configurations and to avoid modifying cryptographic
+options and parameters, or using experimental or deprecated cryptography.
+          </li>
+        </ul>
+
+        <p>
+The guidance above is provided to ensure that useful cryptographic options and
+parameters are provided at the lower layers of the architecture while not
+exposing those options and parameters to application developers that are ill
+suited to understand the benefits and drawbacks of each option.
+        </p>
+
+        <p class="issue"
+          title="Use of experimental and deprecated cryptography">
+The VCWG is seeking guidance on adding language to allow the use of experimental
+or deprecated cryptography. By default, those features will be disabled and will
+require the developer to specifically allow use on a per-cryptographic suite
```

Comment:
Which `developer` here — library or application or other?

Reply:
Good catch... it was meant to be "application", which I've included in the text now.

Reply:
This can be marked "resolved" (but I don't have that button)